### DISCUSSION TITLE
ch4-rpc/ch4-08-grpcurl

### DISCUSSION THREAD

**Discussion by giscus** [chai2010/advanced-go-programming-book#621] [General]
# ch4-rpc/ch4-08-grpcurl

Go语言高级编程

https://chai2010.cn/advanced-go-programming-book/ch4-rpc/ch4-08-grpcurl.html

**Comment by PaiGack** [comment]
postman 也支持 gRPC 了

**Reply by zh7606559342012** [reply]
这么强的吗，好方便
